### PR TITLE
Implement DDL batching

### LINKED ISSUE #22: DDL batching support
`spanner-cli -e" {statements} "` can execute multiple statements, but they are executed sequentially.
If statements are DDLs, it should be executed as a single large batch in practice.
https://cloud.google.com/spanner/docs/schema-updates?hl=en#large-updates

Consideration:
* add a flag like `--batch` or not
* check all statements before execution and batch them only if they are all DDL statements
  * or batch a subsequence of statements that are consecutive DDL statements

### PR DESCRIPTION
**Pull request by apstndb**
~Add `--batch` option which executes DDL statements as batch.~
Updated:
Implement automatic DDL batching if all statements are DDL statements.
Subsequence batching should be possible but it is out of scope of this PR.

- [x] Add tests
  - [ ] ~Add integration tests~
- [x] ~Add validation of flags~

close #22

### CODE REVIEW

**Comment by apstndb** [comment]
I have noticed integration tests is missing.

File: cli.go [hunk 1]
```diff
@@ -375,3 +391,24 @@ func printResult(out io.Writer, result *Result, mode DisplayMode, withStats bool
...
+func buildDdlStatements(input string) Statement {
+	var ddls []string
+	for _, separated := range separateInput(input) {
+		stmt, err := BuildStatement(separated.statement)
+		if err != nil {
+			return nil
+		}
+
+		switch ddl := stmt.(type) {
+		case *DdlStatements:
```
Comment (yfuruyama) [inline cli.go]: Do we need to check `DdlStatements` here although `BuildStatement()` doesn't populate such statement?

File: cli_test.go [hunk 2]
```diff
@@ -57,6 +59,35 @@ func TestSeparateInput(t *testing.T) {
 	}
 }
 
+func TestBuildDdlStatements(t *testing.T) {
+	tests := []struct {
+		Input    string
+		Expected Statement
+		Ok       bool
```
Comment (yfuruyama) [inline cli_test.go]: How is `Ok` used?

Comment (apstndb) [reply cli_test.go]: Fixed.

File: cli.go (hunk 1 above)
Comment (apstndb) [reply cli.go]: It is not needed.
Fixed.

**Comment by apstndb** [comment]
Thanks for reviewing!
I want to skip adding an integration test this time.

**Pull request merged**